### PR TITLE
go-systemd build script

### PR DESCRIPTION
**Pull request by vaibhav-bhadade**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@vaibhav-bhadade looks like you are missing some fields from the script header and hence the basic validation checks are failing on travis, please check

**Comment by vaibhav-bhadade** [comment]
go systemd package cant run in container , it should be tested on VM only

File: g/go-systemd/github.com_coreos_go-systemd_ubi_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+set -e
+
+PACKAGE_NAME=go-systemd
+PACKAGE_VERSION=${1:-48702e0da86bd25e76cfef347e2adeb434a0d0a6}
+GO_VERSION="go1.17.5"
+
+ 
+#install dependencies
+yum install -y wget git tar dbus gcc-c++&& \
```
Comment (kandarpamalipeddi) [inline g/go-systemd/github.com_coreos_go-systemd_ubi_8.4.sh:30]: Looks like, we we need to install `systemd-devel`  package.

Observed below error while running this script on my VM :

```
[root@p006vm76 go-systemd]# go test ./... -v
# github.com/coreos/go-systemd/sdjournal
sdjournal/journal.go:27:11: fatal error: systemd/sd-journal.h: No such file or d                                                                                        irectory
 // #include <systemd/sd-journal.h>
           ^~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

File: g/go-systemd/github.com_coreos_go-systemd_ubi_8.4.sh (hunk 1 above)
Comment (vaibhav-bhadade) [reply g/go-systemd/github.com_coreos_go-systemd_ubi_8.4.sh]: installation for systemd-devel package added

**Comment by Sapana-Khemkar** [comment]
closing as script for latest version is available.

**Pull request closed without merging**